src/components/github-htmlview.component.js

```diff
@@ -48,6 +48,10 @@ const styles = {
   code: codeStyle,
   span: textStyle,
   p: { ...textStyle, margin: 0, padding: 0 },
+  sub: {
+    ...hrStyle,
```

Comment:
I think that you should be using `textStyle` instead

Reply:
Also, this is a new feature, not a bug fix. Could you update your PR description? 🙏